homeassistant/components/webostv/media_player.py

```diff
@@ -61,6 +61,8 @@
 
 LIVE_TV_APP_ID = "com.webos.app.livetv"
 
+ATTR_SOUND_OUTPUT = "sound_output"
```

Comment:
Don't define this twice. Put it a central position in the package and import where needed.